javaparser-core/src/main/java/com/github/javaparser/ast/type/ClassOrInterfaceType.java

```diff
@@ -55,7 +58,8 @@
  *
  * @author Julio Vilmar Gesser
  */
-public final class ClassOrInterfaceType extends ReferenceType implements NodeWithSimpleName<ClassOrInterfaceType>, NodeWithAnnotations<ClassOrInterfaceType>, NodeWithTypeArguments<ClassOrInterfaceType> {
+public final class ClassOrInterfaceType extends ReferenceType<ResolvedReferenceType> implements NodeWithSimpleName<ClassOrInterfaceType>,
```

Comment:
We're trying to get rid of the `<?>` stuff on nodes, I think this will reintroduce one :-(

Reply:
Oh, if you need help with this, I have some strategies.

Reply:
I am all ears :D